### ISSUE TITLE
Nested AttentionSeq2Seq still causes MissingInputException

### ISSUE THREAD

**Issue by phdowling** [farizrahman4u/seq2seq#131]
I think this was an issue before, and may have been partially fixed, but I am facing this problem again.

I am building a nested model that includes an AttentionSeq2Seq, and Theano throws a MissingInputException when I call model.fit()

See runnable gist: https://gist.github.com/phdowling/d9f4b3d2c508ad76bddf53630155fd4c

To verify that seq2seq is the problem, I included a USE_SEQ2SEQ flag that shows that the rest of the model works if seq2seq is not used and only a single output is generated.

**Comment by phdowling** [comment]
I believe I've pinpointed the issue: if I remove the dropout kwarg, this code works. It seems dropout breaks the model. @farizrahman4u could you have a look if you can see what is wrong with the dropout implementation?

**Comment by farizrahman4u** [comment]
Are you using theano backend? 
https://groups.google.com/forum/#!topic/theano-users/8diyZjq6ngc

**Comment by dr-costas** [comment]
See #119

**Comment by phdowling** [comment]
Yes, I am using the Theano backend. However this has not caused MissingInputError in the past for other RNN classes as far as I am aware

**Comment by farizrahman4u** [comment]
Unlike other RNN classes, here the dropout appears inside the scan function.

**Comment by dr-costas** [comment]
Any news on the missing input error?

**Comment by farizrahman4u** [comment]
There is a pr on keras side which fixes this.. waiting for it. TBH, I am quite busy with other stuff.. I would really appreciate if someone could take up a couple of issues..
